contrib/builder/deb/ppc64le/ubuntu-trusty/Dockerfile

```diff
@@ -6,8 +6,12 @@ FROM ppc64le/ubuntu:trusty
 
 RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libsqlite3-dev pkg-config golang-1.6 libsystemd-journal-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
+# Install Go
+# ppc64le doesn't have official go binaries, so use a distro packaged version of go
+# to build go from source.
+# NOTE: ppc64le has compatibility issues with older versions of go, so make sure the version >= 1.6
 ENV GO_VERSION 1.7.3
-ENV GO_DOWNLOAD_URL https://golang.org/dl/go${GO_VERSION}.src.tar.gz
+ENV GO_DOWNLOAD_URL https://storage.googleapis.com/golang/go${GO_VERSION}.src.tar.gz
```

Comment:
Any chance I could convince you to switch all our references to use `golang.org/dl` consistently instead (since it's less likely to require changes in the future, similar to `golang.org/issue/xxx` or `golang.org/cl/nnn` links)? :innocent: